RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -916,7 +916,7 @@ boolean L13_towerNSHedge()
 
 	maximize_hedge();
 	cli_execute("auto_pre_adv");
-	if(!acquireHP())
+	if(!acquireHP(my_maxhp()))
```

Comment:
you also want to heal to full for the walls of bones, meat and skin, and the shadow, all of which do %max HP damage

Reply:
Added an `acquireFullHP()` function to replace all those calls to acquireHP() in the tower.